### ISSUE TITLE
The output of a channel as an input in Extra mixes.

### ISSUE THREAD

**Issue by Woita2095** [FrSky-OS/HORUS-X10-X10S-X12S-bug-report#174]
It is really needed to have the output of a channel as an input in Extra mixes.

I try to explain this:
For example, the very common mix Flaps -> Elevator. The flaps are on a switch and they are slowed down via any method.
And then the Elevator is needed to compensate th Flaps, exactly at the same speed as the Flaps.
In this time it is not possible to synchronize the elevator with the flaps, it is very difficult do it via parameter "speed" in the Extra mix, because every change in flaps take effect in the speed of the elevator, (and the speed of the elevator is different as the speed of the flaps) which is very annoying.

Having the feature "output of a channel as an input in Extra mixes" would solve this. 

Besides, it would solve many other problems, not only the one I described above.

Is it possible to implement such a featue?

**Comment by choochoo2** [comment]
I agree, this seems basic.  Even with my primitive DX8, with a flap to elevator mix, changing the flap speed is automatically matched by the elevator compensation.  Using a second speed adjust to match the first one is unnecessarily cumbersome at best.  Using a channel output as a mixer input seems like a workable solution that may apply other places.
